src/main/java/hudson/plugins/git/UserRemoteConfig.java

```diff
@@ -75,19 +80,22 @@ public String toString() {
     public static class DescriptorImpl extends Descriptor<UserRemoteConfig> {
 
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item project,
```

Comment:
I'm not sure I see the benefit, at least not in the context of the git plugin.  The plugin is widely enough used that the act of making a method public is already enough to place it into the "assumed contract" for the git plugin API.  I've been assuming "if it can be called, act like it is called by someone".

Even with the annotation, I don't see how that will reduce the impact of a change to the method signature.  If the existing signature is called from Jelly EL or from Groovy EL, I think that a signature change will break those callers whether or not the annotation is there.  Have I misunderstood something there?